### PR TITLE
Fix exploits

### PR DESCRIPTION
**Pull request by Apmunute**
patches:
```
/say @e[distance=distance=..99999]
/say @e["distance"=..99999]
/say @e['distance'=..99999]
/execute at @e run forceload add ~ ~ 
/fill ~ 0 ~ ~1 ~1 ~1 command_block{Command:"say test",auto:2b} destroy
/execute as @e run setblock ...
/execute as @e run debug report
/execute as @e run summon minecraft:command_block_minecart ~ ~ ~ {Command:"say test"}
/say @e@e
/datapack
/minecraft:datapack
/execute run datapack
/debug
/ess reload
/ld reload
/paper reload
```

### CODE REVIEW

**Comment by Overcast-Gaming** [comment]
bug.exe

**Comment by business-goose** [comment]
cheebs

**Review by business-goose — APPROVED** [review]

**Review by business-goose — APPROVED** [review]
mhm looks good

**Comment by business-goose** [comment]
yes patch all the exploits you can

**Comment by Apmunute** [comment]
I think I patched all

**Comment by business-goose** [comment]
ima check if the patches work

**Comment by business-goose** [comment]
all patches functional 👍

File: src/main/java/pw/kaboom/extras/modules/server/ServerCommand.java [hunk 1]
```diff
@@ -19,6 +46,28 @@ public static String checkCommand(final CommandSender sender, final String comma
 
 		try {
 			switch (commandName) {
+                case "/minecraft:datapack":
```
Comment (mathiascode) [inline src/main/java/pw/kaboom/extras/modules/server/ServerCommand.java:49]: Remove these, I'm adding them to commands.yml instead.

**Review by mathiascode — CHANGES_REQUESTED** [review]
Change spaces to tabs, otherwise LGTM.

**Review by mathiascode — APPROVED** [review]
Thanks!

**Pull request merged**